analytics/client.go

```diff
@@ -41,9 +41,16 @@ type Client interface {
 // NewClient returns a new Google Analytics client for submitting Wash analytics.
 func NewClient(config Config) Client {
 	if config.Disabled {
-		log.Debugf("Analytics opt-out is set, analytics will be disabled")
+		log.Debug("Analytics opt-out is set, analytics will be disabled")
 		return &noopClient{}
 	}
+
+	// Don't submit analytics for untagged builds
+	if version.BuildVersion == "unknown" {
```

Comment:
I thought about using a more restrictive check, like a regex on semver. But we can do the same thing in Google Analytics and it'll be relatively rare for people to apply non-semver tags. I just do it locally to remember which version I have installed.